src/i18n/locales/en/catalog.json

```
@@ -1061,6 +1061,9 @@
 		},
 		"originalPrice": "Original Price",
 		"billingModel": "Billing Model",
+		"bucketSize": "Bucket Size (Optional)",
```

Biome (2.5.7)

[error] The key bucketSize was already declared.

(lint/suspicious/noDuplicateObjectKeys)